website/docs/d/pi_instance_network.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_instance_network"
+description: |-
+  Retrieve information about a single network attached to a Power Systems Virtual Server instance.
+---
+
+# ibm_pi_instance_network
+
+Retrieve information about a specific network on a Power Systems Virtual Server instance. For more information about Power Virtual Server instances, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
+
+## Example Usage
+
+```terraform
+data "ibm_pi_instance_network" "ds_instance_network" {
+  pi_cloud_instance_id = "49fba6c9-23f8-40bc-9899-aca322ee7d5b"
+  pi_instance_id       = "cea6651a-bc0a-4438-9f8a-a0770b112ebb"
+  pi_network_id        = "52b7c0b1-1df1-495a-9c2d-8b7a6c5ef9aa"
+}
+```
+
+### Notes
+
+- Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+- If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  - `region` - `lon`
+  - `zone` - `lon04`
+  
+Example usage:
+
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+
+## Argument Reference
+
+Review the argument references that you can specify for your data source.
+
+- `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
+- `pi_instance_id` - (Required, String) The unique identifier or name of the instance.
```

Comment:
I think we should move away from mentioning that name is also allowed to avoid another app config issue.
```suggestion
- `pi_instance_id` - (Required, String) The PVM instance id.
```